.github/workflows/release.yml

```diff
@@ -122,14 +122,23 @@ jobs:
         id: extract_branch
         run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
       - name: Invoke Oclif workflow
-        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true'}}
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true' && steps.extract_branch.outputs.branch == 'main'}}
         run: node .github/scripts/trigger-workflow.js
         env:
           WORKFLOW_NAME: 'oclif-release.yml'
           REPO_NAME: twilio/twilio-cli
           REPO_ACCESS_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
           BRANCH_NAME: ${{steps.extract_branch.outputs.branch}}
           INPUTS: '{ "home-brew-branch": "${{github.event.inputs.homebrew-branch}}", "tag-name": "${{needs.release.outputs.tag-name}}", "pre-release": "${{github.event.inputs.homebrew-prerelease}}"}'
+      - name: Invoke Oclif workflow for release candidate
+        if: ${{needs.release.outputs.tag-name != '' && github.event.inputs.draft-release != 'true' && steps.extract_branch.outputs.branch != 'main'}}
```

Comment:
Will we have the branch name as`release-feature branch` always? 
If yes, we could've probably used condition `steps.extract_branch.outputs.branch == 'release-feature-branch'`. Will that make a difference?

Reply:
No, Every time the branch name will change. In future we might have different name for breaking changes branch and wanted to do a release candidate. So they can create a branch on top of `main` and do their changes and release a release candidate.